app/connector/UserDetailsConnector.scala

```diff
@@ -32,20 +30,22 @@ trait UserDetailsConnector {
   def getUserDetails(userDetailsUri: String)(implicit hc: HeaderCarrier) = http.GET[UserDetails](userDetailsUri)
 }
 
-object CredentialRole extends Enumeration {
-  type CredentialRole = Value
-  val User, Unknown = Value
+case class CredentialRole(value: String) extends AnyVal {
```

Comment:
We don't need to know what the credentialRole is. 
We only need to know if it is an admin. 
Before our change it was like you described with 2 groups User and Unkonwn. 
We changed it because we need to audit the credential role, but we don't want to care about it otherwise.